### ISSUE TITLE
Stop printing stacktraces to my console

### ISSUE THREAD

**Issue by prestongarno** [graphcool/graphcool-cli#143]
It's completely useless because it doesn't tell me what went wrong

**Comment by timsuchanek** [comment]
That's a very good point. But do you even see stack traces without setting the `DEBUG` environment variable?

**Comment by prestongarno** [comment]
Yeah, although I just realized that I was using the argument `--header` from get-graphql-schema.  I'm really just trying to do schema introspection of open graphql API endpoints for testing my client library. How do I set headers with auth tokens from the CLI?
